Utilities/ShmManager/src/ShmManager.cxx

```diff
@@ -75,7 +75,6 @@ struct ShmManager {
       }
       uint16_t id = stoi(conf.at(0));
       uint64_t size = stoull(conf.at(1));
-      segmentCfgs.emplace_back(fair::mq::shmem::SegmentConfig{id, size, "rbtree_best_fit"});
```

Comment:
Please don't change this file. It has nothing to do with your feature and just taking it out is not a solution.
If you have problem with this file, please open a JIRA ticket describing the issue.

Reply:
Sorry for that. I revoked those changes in newest commits.